lib/pact/provider/configuration/pact_verification_with_tags.rb

```diff
@@ -0,0 +1,54 @@
+require 'pact/provider/pact_verification_with_tags'
+require 'pact/provider/pact_uri'
+require 'pact/shared/dsl'
+require 'pact/provider/world'
+
+module Pact
+  module Provider
+
+    module Configuration
+
+      class PactVerificationWithTags
```

Comment:
I think this class is about fetching pacts from the broker, not about tags specifically (the user may not specify any tags at all), so I reckon it should be called something like PactVerificationFromBroker